### DISCUSSION TITLE
Function to calculate the new coordinates from given base coordinates, along with a distance & a bearing?

### DISCUSSION THREAD

**Discussion by mc1903** [NetTopologySuite/NetTopologySuite#640] [Q&A]
Hello,

Is there a function in NetTopologySuite that for a given set of base coordinates, along with a distance, a bearing (?azimuth?) and/or an optional altitude; will calculate the new end point coordinates?

Any advice/examples would be appreciated.

Thanks
M

**Comment by airbreather** [comment]
Not that I know of in NetTopologySuite.

If you can assume a sphere instead of an ellipsoid, the formulas over here are what you need: https://www.movable-type.co.uk/scripts/latlong.html#dest-point

To handle altitude, just add it to the radius.

**Reply by mc1903** [reply]
Sorry for delay in responding. Thank you for your help.
